backend/de.metas.externalsystem/src/main/sql/postgresql/system/80-de.metas.externalsystem/5621220_sys_gh12312_BPartnerLookupVia_MetasfreshId_Description.sql

```diff
@@ -0,0 +1,24 @@
+-- 2022-01-12T15:59:18.848Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+UPDATE AD_Ref_List SET Description='Abgleich mit der metasfresh-ID (C_BPartner_ID) des Geschäftspartners. Bei der Verwendung dieser Abgleicht-Art wird während der BPartner-Synchronisierung kein Kontakt übermittelt und somit erscheint keine Kontakt-ID in den importierten Auftragskandidaten.',Updated=TO_TIMESTAMP('2022-01-12 17:59:18','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Ref_List_ID=543069
+;
+
+-- 2022-01-12T15:59:30.145Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+UPDATE AD_Ref_List_Trl SET Description='Abgleich mit der metasfresh-ID (C_BPartner_ID) des Geschäftspartners. Bei der Verwendung dieser Abgleicht-Art wird während der BPartner-Synchronisierung kein Kontakt übermittelt und somit erscheint keine Kontakt-ID in den importierten Auftragskandidaten.',Updated=TO_TIMESTAMP('2022-01-12 17:59:30','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Language='de_CH' AND AD_Ref_List_ID=543069
+;
+
+-- 2022-01-12T15:59:32.730Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+UPDATE AD_Ref_List_Trl SET Description='Abgleich mit der metasfresh-ID (C_BPartner_ID) des Geschäftspartners. Bei der Verwendung dieser Abgleicht-Art wird während der BPartner-Synchronisierung kein Kontakt übermittelt und somit erscheint keine Kontakt-ID in den importierten Auftragskandidaten.',Updated=TO_TIMESTAMP('2022-01-12 17:59:32','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Language='de_DE' AND AD_Ref_List_ID=543069
```

Comment:
```suggestion
UPDATE AD_Ref_List_Trl SET Description='Abgleich mit der metasfresh-ID (C_BPartner_ID) des Geschäftspartners. Bei der Verwendung dieser Abgleich-Art wird während der BPartner-Synchronisierung kein Kontakt übermittelt und somit erscheint keine Kontakt-ID in den importierten Auftragskandidaten.',Updated=TO_TIMESTAMP('2022-01-12 17:59:32','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Language='de_DE' AND AD_Ref_List_ID=543069
```

there is a typo in my trl..